packages/mold-bin/mold-bin.pacscript

```diff
@@ -0,0 +1,22 @@
+#  __      __________   ______
+# /  \    /  \_____  \ /  __  \
+# \   \/\/   //  ____/ >      <
+#  \        //       \/   --   \
+#   \__/\  / \_______ \______  /
+#        \/          \/      \/
+maintainer="wizard-28 <wiz28@pm.me>"
+
+name="mold-bin"
+version="1.11.0"
+arch=("amd64")
+url="https://github.com/rui314/mold/releases/download/v${version}/mold-${version}-x86_64-linux.tar.gz"
+description="A modern linker"
+breaks=("${name}-git" "$name")
+hash="bf788940db4a9ac19e7745c821bf6ee18ff4d75441a803d84f86c9f3b0aa2a5e"
+repology=("project: mold")
+
+install() {
+	sudo mkdir -p "${pkgdir}/usr/local"
+	sudo mv ./* "${pkgdir}/usr/local"
```

Comment:
Doesn't debian and arch forbid packaging to /usr/local?

Reply:
Pretty sure it is.

Reply:
Ok, suggest the change

Reply:
I don't know where it should go. I just know it can't be `/usr/local/`

Reply:
Found it: https://www.debian.org/doc/packaging-manuals/fhs/fhs-3.0.html#usrlocalLocalHierarchy